src/components/pages/lessons/overlay/go-pro-cta-overlay.tsx

```diff
@@ -210,10 +226,18 @@ const GoProCtaOverlay: FunctionComponent<JoinCTAProps> = ({lesson}) => {
             </div>
           )}
           <button
-            className="px-10 py-4 mt-5 font-medium text-center text-white transition-all duration-300 ease-in-out bg-blue-600 rounded-md hover:bg-blue-700 hover:scale-105"
+            className={`px-10 py-4 mt-5 h-[60px] font-medium flex justify-center items-center text-white transition-all duration-300 ease-in-out bg-blue-600 rounded-md hover:bg-blue-700 hover:scale-105 ${
+              pricesLoading
+                ? 'opacity-60 cursor-default'
+                : 'hover:bg-blue-700 hover:scale-105'
+            }`}
```

Comment:
Borrowed these style changes from something very similar that @DrShpongle did in the Pricing Widget.